src/read/elf/file.rs

```diff
@@ -116,6 +116,7 @@ where
             elf::EM_386 => Architecture::I386,
             elf::EM_X86_64 => Architecture::X86_64,
             elf::EM_MIPS => Architecture::Mips,
+            elf::EM_MIPS_RS3_LE => Architecture::Mips,
```

Comment:
This one probably needs a new `Architecture` variant.

Reply:
I've noticed that `llvm-objdump` displays ELFs of type `EM_MIPS_RS3_LE` as "file format ELF32-unknown", so maybe in practice it isn't used and should instead be removed? `mipsel` can presumably also be detected via the ELF's endianness.

Reply:
How did you generate a file with `EM_MIPS_RS3_LE`?

Reply:
With this crate – originally the `write` module used `EM_MIPS` for big-endian and `EM_MIPS_RS3_LE` for little-endian. Then I saw that the `llvm-objdump` didn't like that, and that `clang --target=mipsel-unknown-none` would also use `EM_MIPS`.

Reply:
Removed this part